docs/contributing.rst

```diff
@@ -27,6 +27,7 @@ Alternatively, you can also use the `requirements.txt` file to install the
 dependencies yourself. That would be::
 
      pip install -r requirements.txt
+     pip install .
```

Comment:
Then, the best way to do this would be to do `pip install -e .`, so that it's editable in place.

Reply:
You are right!
I have modify it in order to reflect your advice.